python/eos/plot/plotter.py

```diff
@@ -789,6 +789,10 @@ class Constraint(BasePlot):
            Must identify one of the observables known to EOS; see `the complete list of observables <../observables.html>`_.
            This is only mandatory in multivariate constraints, since these can constrain more than one observable simultaneously.
 
+        The following keys are optional:
+
+         * ``rescale-by-width`` (*bool*) -- Rescales binned constraints by the bin width. Defaults to false.
```

Comment:
I think we should be more specific here. Does it rescale by the inverse of the bin width or the bin width? I think it is the former, to turn integrated measurements in averaged measurements, making it possible to show integrated measurements in a differential plot.

Reply:
Better?

Reply:
Definitely! Can you also add something like: `This is often required to compare theory (integrated) predictions and experimental (averaged) measurements.`

Reply:
Done